di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java

```diff
@@ -161,6 +161,7 @@ private void saveOrUpdateCimitStubItems(String userId, List<CimitStubItem> cimit
                         cimitStubItemService.persistCimitStub(
                                 userId,
                                 cimitStubItem.getContraIndicatorCode().toUpperCase(),
+                                cimitStubItem.getIssuers(),
```

Comment:
Do we need to update `mapToContraIndications` to set this? Otherwise I think it'll always be null...